revscoring/features/tests/test_feature.py

```diff
@@ -7,17 +7,27 @@
 from ...dependent import solve
 from ..feature import Feature
 
-five = Feature("five", lambda: 5, returns=int, depends_on=[])
+
+def return_five(): return 5
+
+five = Feature("five", return_five, returns=int, depends_on=[])
 
 def identity_process(value): return value
 int_identity = Feature("int_identity", identity_process,
                                        returns=int, depends_on=["value"])
 
+def check_feature(feature, expected):
+
+    eq_(feature.returns, type(expected))
+
+    eq_(hash(pickle.loads(pickle.dumps(feature))), hash(feature))
```

Comment:
Won't this use of `hash` have problems such as http://stackoverflow.com/a/793835/2062663 ?